### ISSUE TITLE
Allow Unlimited Size of Tags & Let Users Define Column Width

### ISSUE THREAD

**Issue by tripluoffthoranlz** [frianasoa/Ze-Notes#8]
Hi,

I need to be able to make some fairly **long** tags - ideas that occur to me which I may or may not use, and don't really boil down to two or three words.

I've done that with one academic work this week - using PRETTY long tags - and the result as to break the table.  

That is, the columns all stayed a fixed width, but the first row with the tags became **very tall**.

And. there was no way to get to the second row with the quotes.  

Basically, it was a full screen showing ONLY VERY TALL COLUMNS of **just** the tags row.  You could scroll up and down that tall tags row, but you could not get to the highlight themselves.

Please can you enable the tags to have no limit on characters?  

And, please could you allow users to define the width of columns?

Sincerely.

Triplu

**Comment by frianasoa** [comment]
Hi!

You could try version 0.4.1-beta if you are in a hurry. I need to do more test before allowing automatic updates, so you need to download and install. You will need to set column display (order, sort, etc) again as I changed the way settings are saved.

https://github.com/frianasoa/Ze-Notes/releases/download/v0.4.1-beta/zenotes-v0.4.1-beta.xpi 

Best.

**Comment by tripluoffthoranlz** [comment]
Hi!!

Will that work with Zotero 6?

Sincerely,

Triplu

**Comment by frianasoa** [comment]
Hi!
It works on Zotero 6 but do not update the plugin just yet unless you just want to test and downgrade back. It is in beta and I am still figuring out some issues. I am a little busy so It might take a couple of weeks until a stable version is out.
Best

**Comment by frianasoa** [comment]
Hi!
The latest version supports setting column width now. I do not think there is a length limit for tags but this should fix the display. 
Head to ZeNotes > Settings and set "Header size" for individual column.
![image](https://github.com/frianasoa/Ze-Notes/assets/5507310/a5e2073d-ed26-4eac-b2a7-9f7037a11622)

You can also change the size of all columns with the  general setting "Header size" below.
![image](https://github.com/frianasoa/Ze-Notes/assets/5507310/704056ab-bf71-4637-9914-22e7952d3008)

If you check "Vertical table", columns will become row and "Header size" will apply to the row height. That is why it is it was renamed "Header size" instead of column width.
